### DISCUSSION TITLE
How to use the python-fasthtml package

### DISCUSSION THREAD

**Discussion by james0r** [AnswerDotAI/fasthtml#191] [Q&A]
I'm new to python but I used pip3 to successfully install the `python-fasthtml` package. When I go to execute the demo file though  I get
```
λ  ~/Downloads  python3 main.py
Traceback (most recent call last):
  File "/Users/jamesauble/Downloads/main.py", line 1, in <module>
    from fasthtml.fastapp import *
ModuleNotFoundError: No module named 'fasthtml'
```
how do imports in python work...and/or what am I doing wrong?

**Comment by TirthNotFoundthedev** [comment]
Did you install correct fasthtml module?
Please make sure you installed it using `pip install python-fasthtml`
If you did not install the correct module (`python-fasthtml`) then follow the following steps:
1. Type: `pip uninstall fasthtml` <- Here enter the name you entered when downloading.
2. Then type `pip install python-fasthtml` in order to donwload the correct module.

If you had never installed the module then use step 2.

Hope this helps you.

**Reply by james0r** [reply]
I installed the right package, but where is it on the file system? is it installed globally? if so how does the python file reference it?

**Reply by bryanwhiting** [reply]
is it an environment issue? `pip list | grep fasthtml`

**Reply by james0r** [reply]
```
λ  ~/Downloads  pip list | grep fasthtml
zsh: /opt/homebrew/bin/pip: bad interpreter: /opt/homebrew/opt/python@3.11/bin/python3.11: no such file or directory
python-fasthtml  0.2.1
```
